lib/Board.rb

```diff
@@ -0,0 +1,38 @@
+class Board
+    def example_board
+        b= [[1,2,3],[4,5,6],[7,8,9]]
+        b
+    end
+
+    def addSpace (c,n) 
+        space = ""
+        for a in 0..n-1
+            space += c 
+        end
+        space
+    end
+    def print_board(arr, cl)
+        if arr.nil?
+            arr = [[1,2,3],[4,5,6],[7,8,9]]
+        end
+        row = arr.length
+        col = arr[0].length
+        
+        cellLength = cl
+        for i in 0..row-1
+            line = ""
+            if i>0&&i<row
+                puts "#{addSpace("-",cellLength)}"+"+"+"#{addSpace("-",cellLength)}"+"+"+"#{addSpace("-",cellLength)}"
```

Comment:
no “puts” and “get.chomp” inside class only Game logic

Reply:
Fixed in next commit